client/asset/dcr/dcr.go

```diff
@@ -938,7 +938,7 @@ func (dcr *ExchangeWallet) tryFund(utxos []*compositeUTXO, enough func(sum uint6
 
 		okUTXOs := make([]*compositeUTXO, 0, len(utxos)) // over-allocate
 		for _, cu := range utxos {
-			if cu.confs >= minconf {
+			if cu.confs >= minconf && cu.rpc.Spendable {
```

Comment:
This is just extra cautious though, right? Should never get this far with a non-Spendable?

Reply:
Yes, extra.  unspendable shouldn't make it up here.